macaroon/Classes/Views/ListComponents/CollectionCell.swift

```diff
@@ -41,3 +51,14 @@ open class CollectionCell<ContextView: ViewComposable>: UICollectionViewCell, Li
         contextView.prepareForReuse()
     }
 }
+
+extension CollectionCell {
+    private func addSeperatorView() {
```

Comment:
Let's make It `open` and return separator view. That way if one wants to make further changes on the view, it should override the method and get separator view from super and modify it.

Reply:
Don't forget @discardableResult since we don't need it in default configuration.